extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -595,8 +534,7 @@ protected function getSearchConfigFromRequest() {
 			->setInterWiki               ( $this->isCorporateWiki() )
 			->setVideoSearch             ( $request->getVal( 'videoSearch', false ) )
 			->setFilterQueriesFromCodes  ( $request->getVal( 'filters', array() ) )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "filters" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)